src/main/java/com/Feature1/WSFeaure1/User.java

```diff
@@ -0,0 +1,39 @@
+package com.Feature1.WSFeaure1;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+
+@Entity
+@Table(name = "User")
```

Comment:
@Xeniia-droid If you are fine with the default table name provided by hibernate (name of the class with first letter capitalized) then you can omit this annotation.